cypress/e2e/models/administration/credentials/credentialsSourceControlUsername.ts

```diff
@@ -33,20 +33,24 @@ export class CredentialsSourceControlUsername extends CredentialsSourceControl {
         inputText(usernameInput, this.username);
     }
 
-    protected fillPassword() {
-        inputText(passwordInput, this.password);
+    protected fillPassword(withHash = false) {
```

Comment:
Modifying this method just for one test case doesn't look right . Let's send the password from test with # in it.